### ISSUE TITLE
The Revive UI isn't showing!

### ISSUE THREAD

**Issue by Kookoogala** [LibreVR/Revive#709]
The UI for the revive in the steamVR menu isn't there it's just plank. Although the tab is showing. And for some reason I can still launch applications via the revive tab but clicking randomly at in the box.
I also had this issue occasionally with advanced settings.
I have tried most of the fixes in the issues tab, but none have worked so far. I.e: reinstalling steamVR and revive, deactivation my second graphics card. But nothing

**Comment by chubschubs** [comment]
Are you running on a laptop? I have been having the same issue but I think I have it half figured out. So if you are running an Nvidia card, open the Nvidia control panel and then select manage 3D settings and then select program settings. Select Revive Overlay from the program list or add it. Then select your dedicated GPU instead of your integrated. Now I am at the point where it shows up but not all of the games do... But it's a start.

**Comment by chubschubs** [comment]
So completely removing Oculus home and reinstalling seemed to resolve the issue with games not showing up in the dashboard.

**Comment by Armada651** [comment]
Interesting, then this might be a case of Revive rendering on the integrated card and SteamVR on the dedicated card. Meaning when SteamVR gives the final UI texture to SteamVR it fails to display it since it's not present on the dedicated card.

**Comment by mapotter99** [comment]
@Kookoogala, did you ever get this issue resolved? 

I tried @chubschubs solution, but it didn’t work.

**Comment by Kookoogala** [comment]
Yeah I did, I just had to reinstall SteamVR and the latest version of
Revive :)

**Comment by mapotter99** [comment]
Darn... already tried both of those solutions as well!  Thanks.

**Comment by mapotter99** [comment]
I've determined the source of this problem for my occurrence... Windows 10's HDR & Advanced Color setting.  I turned this off, and the UI for both Revive and Advanced Settings displayed properly.

I recently bought a new 4K HDR TV for my basement, which also serves as the monitor for my gaming PC.  I just happened to think to try disabling HDR & Advanced Color in the Windows 10 Display options.

HOORAY!
